src/api/services/lectureService.ts

```
@@ -0,0 +1,486 @@
+    
+    // Extract evaluation items - look for table with 항목/비중/만점/공개여부 headers
+    const evaluationItems: EvaluationItem[] = [];
+    const evalTable = Array.from(doc.querySelectorAll('table')).find(table => 
+      (table.textContent?.includes('항목') && table.textContent?.includes('비중')) ||
+      table.textContent?.includes('성적평가항목')
+    );
+    
+    if (evalTable) {
+      // Get ALL rows, not just tbody rows
+      const allRows = evalTable.querySelectorAll('tr');
+      let evaluationRowCount = 0;
```

ESLint

[error] 'evaluationRowCount' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)